### DISCUSSION TITLE
맵 내부에서 크리에이터 코스튬 구매와 입어보는 기능 구현 문의

### DISCUSSION THREAD

**Discussion by SweetPotato7432** [naverz/zepeto-studio-kor#938] [5. Q&A]
맵 내부에서 옷을 입고 구매까지 가능한 상점 맵을 만들고 싶어 문의 드립니다.
코스튬 혹은 특정 아이템을 착장 및 구매하여 다른 맵까지 사용 가능한 맵을 만드려고 하는데 (EX: 한강공원 나이키 스토어)
이 부분을 지원 혹은 구현 가능 한지 궁금합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 현재는 코스튬 아이템은 제페토 앱 내부 상점에서만 구입할 수 있습니다.

코스튬 아이템을 유니티 월드 내에서 판매가 가능하게 하기 위해서는 연동 작업부터 정책까지 고려할 내용이 많아서
지원 준비 중에 있으나 시간이 많이 소요되는 점 양해 부탁 드립니다.
감사합니다.
